.clusterfuzzlite/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -eu
+# Supply build instructions
+# Use the following environment variables to build the code
+# $CXX:               c++ compiler
+# $CC:                c compiler
+# CFLAGS:             compiler flags for C files
+# CXXFLAGS:           compiler flags for CPP files
+# LIB_FUZZING_ENGINE: linker flag for fuzzing harnesses
+
+# Copy all fuzzer executables to $OUT/
+
+# Copy all fuzzer executables to $OUT/
+$CXX $CFLAGS $LIB_FUZZING_ENGINE \
+  $SRC/fast-cpp-csv-parser/.clusterfuzzlite/parse_fuzzer.cpp \
```

Comment:
Is it a good idea to add ubsan and or asan here?

Reply:
what about thread sanitizer?

Reply:
`CXXFLAGS` controls which sanitizer to enable. The sanitizer matrix possible is defined in the GitHub action:

```yaml
    strategy:
      fail-fast: false
      matrix:
        sanitizer: [address]
```

At the moment I've only enabled ASAN, but we can add all three: `address/undefined/memory`, see [here](https://google.github.io/clusterfuzzlite/running-clusterfuzzlite/github-actions/#pr-fuzzing) for details.